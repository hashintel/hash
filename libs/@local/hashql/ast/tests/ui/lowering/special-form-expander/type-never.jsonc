//@ run: pass
//@ description: Never type should be properly lowered
["::kernel::special_form::type", "x", "::kernel::type::Never", "x"]
