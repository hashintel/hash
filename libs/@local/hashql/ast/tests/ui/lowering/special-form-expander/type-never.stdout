Expr#4294967040@24
  ExprKind (Type)
    TypeExpr#4294967040@24 (name: x)
      Type#4294967040@19
        TypeKind (Never)
      Expr#4294967040@23
        ExprKind (Path)
          Path#4294967040@23 (rooted: false)
            PathSegment#4294967040@22 (name: x)
